Makefile

```diff
@@ -4,6 +4,9 @@ IMG ?= controller:latest
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 ENVTEST_K8S_VERSION = 1.25.0
 
+# TEST_COVER_OUT determines path for the output file with coverage 
+TEST_COVER_OUT ?= $(shell pwd)/cover.out
```

Comment:
Should you add it to git ignore if run locally?

Reply:
It is in .gitignore 
https://github.com/kyma-project/warden/blob/main/.gitignore#L16